presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/PackageAPI.java

```diff
@@ -128,7 +128,8 @@ public synchronized Response upgrade(@ApiParam("Url to fetch package") String ur
             LOGGER.error("Invalid url: {}", urlToFetchPackage, e);
             return Response.status(BAD_REQUEST).entity("Invalid url").build();
         }
-        return Response.status(ACCEPTED).entity("Presto is being upgraded.").build();
+        return Response.status(ACCEPTED).entity("Presto is being upgraded. " +
+                "Please check back later using the status api to make sure that upgrade succeeded.").build();
```

Comment:
To be consistent with the airlift PR, put replace the space at the end of the first string with `\r\n`, and use this message:
> To verify that the upgrade succeeded, check back later using the status API.